gamification/activity.py

```diff
@@ -0,0 +1,30 @@
+from gamification.labels import used_labels
+
+
+def get_activity(activity_type, labels):
+    """
+    Get an activity based on the labels on issues and mrs.
+
+    :param activity_type: type of the activity. e.g. issue
+                          or merge_request
+    :param labels: a QuerySet dict containing the 'name'
+                   as key and 'name of the label' as value
+    :return: a touple of points and activity string
+    """
+    labels_list = [label['name'] for label in labels]
+
+    # Removing labels which are not in used_labels
+    for label in labels_list[:]:
+        if label not in used_labels():
+            labels_list.remove(label)
+
+    # Sorting the labels to get consistent activity
+    labels_list.sort()
+    if activity_type == 'issue':
+        activity = 'Created a '
+    elif activity_type == 'merge_request':
+        activity = 'Solved a '
+    for label in labels_list:
+        activity = activity + label + ' '
+    activity = activity + 'issue'
```

Comment:
this will append 'issue' to merge_request activities ?

Reply:
Yes, `merge_request` or `issue` activities both.

If the activity is merge_request, then it would become:
```
 Solved a area/documentation difficulty/medium importance/medium issue
```
And if the activity is `issue`, then it would become:
```
 Created a area/documentation difficulty/medium importance/medium issue
```